src/app/AppCore.h

```diff
@@ -167,6 +167,10 @@ struct GpuTimerInfo {
   uint64_t end_time;
 };
 
+struct CASUpscalingInfo {
```

Comment:
nit: could "parameters" or "params" or something to that a affect be more clear than "info" perhaps?

Reply:
I agree with Nick. CASUpscalingParams or CASDynamicParams would be cool.

Reply:
Done, thanks!